pkg/cache/groupcache.go

```diff
@@ -59,13 +60,17 @@ type GroupcacheConfig struct {
 
 	// How often we should resolve the addresses.
 	DNSInterval time.Duration `yaml:"dns_interval"`
+
+	// Timeout specifies the read/write timeout.
+	Timeout time.Duration
```

Comment:
Should we: 1) prefer `model.Duration` here?; 2) add `yaml` tag?

Reply:
All of the other providers use `time.Duration` so I used the same here. We should probably stick to `model.Duration` in code which is somehow related to PromQL? :shrug:  I'll add a yaml tag to be explicit, good point :+1:

Reply:
Seems you're right, we use `time.Duration` for caching YAML configs, although in other places in the code (e.g. object store configs) we are using `model.Duration`, which is a bit confusing. I thought we leverage `model.Duration` to keep the duration parsing logic consistent across all configs, but it's not true.